gprofiler/profilers/python.py

```diff
@@ -249,7 +249,9 @@ def _select_processes_to_profile(self) -> List[Process]:
         if is_windows():
             all_processes = [x for x in pgrep_exe("python")]
         else:
-            all_processes = [x for x in pgrep_maps(DETECTED_PYTHON_PROCESSES_REGEX)]
+            all_processes = [
+                x for x in pgrep_maps(r"(^.+/(lib)?python[^/]*$)|(^.+/site-packages/.+?$)|(^.+/dist-packages/.+?$)")
```

Comment:
I merged - so update the granulate-utils reference to https://github.com/Granulate/granulate-utils/commit/cb66a9fe803d0448ac6eea6aceb85e34a88b0b8f, and revert this change